stock_picking_invoice_product_group/models/account_invoice.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Alex Comba - Agile Business Group
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, fields
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    related_picking_ids = fields.Many2many(
+        comodel_name='stock.picking',
+        relation='account_invoice_stock_picking_rel',
```

Comment:
v8 don't need relation, column1, column2 variables.
You can use
`related_picking_ids = fields.Many2many('stock.picking', string='Related Pickings'`